CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ SET(MAJOR_VERSION 1)
 IF ( NOT VERSION )
    IF ( "${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "Windows" )
       message("windows: Extracting git software version")
-      execute_process(COMMAND cmd /c "git rev-list --branches HEAD | find /v "" /c" OUTPUT_VARIABLE GIT_VERSION WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+      execute_process(COMMAND cmd /c "git rev-list --branches HEAD | find /v " " /c" OUTPUT_VARIABLE GIT_VERSION WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
```

Comment:
Or even better:

```
execute_process(COMMAND git rev-list --count HEAD OUTPUT_VARIABLE GIT_VERSION WORKING_DIRECTORY ${CMAKE_SOURCE_DIR} OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Reply:
What's wrong with the way it's currently done. Seems to be working on the platforms I've tested it on

Reply:
Yep, seems to work in practice aside from the end of line difference¹. I'm not sure _how_ it works because it didn't seem to do the same thing when I ran it on the command line. The difference with the version I proposed is it performs the count within git itself so it would be a bit faster. In practice I'm not sure it's observable.

¹ Adding the `OUTPUT_STRIP_TRAILING_WHITESPACE` option would resolve the difference.